src/Microsoft.Identity.Web/MergedOptions.cs

```diff
@@ -236,6 +237,14 @@ internal static void UpdateMergedOptionsFromMicrosoftIdentityOptions(MicrosoftId
 
             mergedOptions.UseTokenLifetime = microsoftIdentityOptions.UseTokenLifetime;
             mergedOptions._confidentialClientApplicationOptions = null;
+
+            foreach (var scope in microsoftIdentityOptions.Scope)
```

Comment:
@jennyf19 
Thinking more, we need to decide if we want to add to the configuration or replace the configuration.

For instance, if by default, the scopes as set by ASP.NET Core are "openid profile offline_access", but a customer doesn't want to have offline_access and profile, do we want to be able to say

```CSharp
services.Configure<MicrosoftIdentityOptions>(OpenIdConnectDefault.AuthenticationScheme, options =>
{
 options.Scopes.Clear();
 options.Scopes.AddRange(new [] {"openid email" })
});
```

as with this current proposed implementation, we would get "openid profile offline_access email"
whereas the developer might intend only "openid email" 

This also applies to any collection in MergedOptions.